### PR TITLE
feat: enable tests on java 10 and 11

### PR DESCRIPTION
**Pull request by lili2311**
- [x] Ready for review
- [x] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Enable testing on java 10 & 11

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,50 +1,111 @@
...
-    - stage: npm release
-      node_js: "8" # This *has* to be the "build leader"
-      script: skip
-      after_success:
-        - npx semantic-release
-branches:
-  only:
-    - master
+  - stage: Node versions + actual Gradle version
+    node_js: 6
```
Comment (dkontorovskyy) [inline .travis.yml]: But you removed node_js 6 and 10, would this work?

File: .travis.yml [hunk 2]
```diff
@@ -1,50 +1,111 @@
...
+    node_js: 8
+    env: GRADLE_VERSION=5.3.1 JDK=10
+    script:
+      - source install-jdk.sh --feature $JDK
+      - ln -sf /etc/ssl/certs/java/cacerts $JAVA_HOME/lib/security/cacerts
+      - sdk install gradle $GRADLE_VERSION
+      - java -version
+      - npm test
+  -
+    node_js: 10
```
Comment (dkontorovskyy) [inline .travis.yml]: But you removed node_js 6 and 10, would this work?

File: .travis.yml (hunk 1 above)
Comment (miiila) [reply .travis.yml]: Yeah, Travis is... creative :-) https://travis-ci.org/snyk/snyk-gradle-plugin/builds/566973238

File: .travis.yml (hunk 2 above)
Comment (miiila) [reply .travis.yml]: dtto

File: .travis.yml (hunk 1 above)
Comment (dkontorovskyy) [reply .travis.yml]: Looks super weird that it picked up that versions from ... nowhere :)

**Comment by lili2311** [comment]
https://github.com/snyk/snyk/pull/651

**Pull request merged**